bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Component.java

```diff
@@ -64,6 +64,19 @@ default String getId() {
     default ComponentData getData() {
         return null;
     }
+    
+    /**
+     * Returns the style system information associated with the component
+     *
+     * @return CSS classes selected by the content author delimited using a SPACE character
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.20.0
+     */
+    @Nullable
+    @JsonProperty("addlCssClassNames")
+    default String getStyleSystemClasses() {
```

Comment:
I would name this just `getAppliedStyleClasses`. Also, I would rather we return a list here instead of a string, it would make checking if a certain class is set so much easier.

Reply:
Done. Please review the updated code and mark this as Resolved if you are satisfied. Thank you !

Reply:
@vladbailescu - Changed the method signature to match with existing ComponentStyleInfo.class behavior. List is replaced with String. Reason - ComponentStyleInfo.class#getAppliedStyleClasses() doesn't return list of strings, but returns list of StyleInfo beans. So parsing them again will duplicate the logic. So used the existing string method as-is. If you feel otherwise please let me know. Thank you !